docs/architecture/resilience.txt

```diff
@@ -114,6 +114,14 @@ Here are some considerations:
    rare consistency or data-loss issues you might encounter when CrateDB
    experiences network or hardware failures (see next section).
 
+
+Nightly Jepsen tests for dirty reads, lost updates, version divergence
```

Comment:
I would be so specific in what the jepsen suite is testing. this could change anytime and then we'd have update this document always..

Reply:
I think we should be specific and it's ok to update this. Otherwise we can drop the entire paragraph.

Reply:
Then lets remove it, I don't agree on being so specific here ;)

Reply:
I like that we mention the Jepsen tests which somehow shows to our users that we care about the resiliency.
On the other hand I also don't like the specific mention of the tests. Personally I would keep it more abstract, but also I wouldn't like to remove the whole paragraph :-(

Reply:
Removed the test details.